ament_cmake_nose/cmake/ament_add_nose_test.cmake

```diff
@@ -133,5 +133,3 @@ function(_ament_add_nose_test testname path)
     LABELS "nose"
   )
 endfunction()
-
-# TODO provide function to register all found tests separately
```

Comment:
The idea was to introspect nosetest and register each individual test function as a separate CTest. I don't think we will purpose the idea though since that make reporting the results in separate JUnit files much more difficult and the runtime would be worse since each test would use a separate Python interpreter.